rclcpp/src/rclcpp/node_interfaces/node_base.cpp

```diff
@@ -216,7 +215,8 @@ NodeBase::get_shared_rcl_node_handle() const
 }
 
 rclcpp::CallbackGroup::SharedPtr
-NodeBase::create_callback_group(rclcpp::CallbackGroupType group_type)
+NodeBase::create_callback_group(
```

Comment:
This line break is not required. It was introduced in the demo by the second argument RealTimeClass which is not used here.